### ISSUE TITLE
Unable to send mail

### ISSUE THREAD

**Issue by TheMadRuffian** [LukeSmithxyz/emailwiz#23]
I followed every step in Luke's latest video on setting up the email server but I couldn't send email at all although I can receive email. I have tried to send to several providers like Gmail, Outlook and Posteo. Even https://appmaildev.com can't detect any email that had been sent.

**Comment by TheMadRuffian** [comment]
It was installed on a Debian 10 Vultr VPS and hosting from Epik.

**Comment by LukeSmithxyz** [comment]
So nothing in spam either?

What's in your system logs?

**Comment by jamzattack** [comment]
This is a duplicate of #21 

You need to contact vultr and request that  port 25 be unblocked.

**Comment by TheMadRuffian** [comment]
> So nothing in spam either?
> 
> What's in your system logs?

There was nothing in the spam folder. The system log says "Connection timed out" for every email provider I had tried previously.

> This is a duplicate of #21
> 
> You need to contact vultr and request that port 25 be unblocked.

I'll try contacting them

**Comment by TheMadRuffian** [comment]
Update: After contacting Vultr, they informed that SMTP ports are being blocking by default to prevent any possible abuse. After clarifying the intentions, they unblocked port 25 and now I can send emails.
